docker-compose-dev.yml

```diff
@@ -6,6 +6,7 @@ volumes:
 services:
   backend:
     image: ghcr.io/envelope-zero/backend:v4.2.2
+    user: root
```

Comment:
This is require because the image now uses a non-root user, but the filesystem is owned by root.

I'm working on a better solution for this, but the effect of setting `user: root` is that it works like with backend versions `v3.x.x`